src/mcpadapt/smolagents_adapter.py

```diff
@@ -3,12 +3,60 @@
 SmolAgents do not support async tools, so this adapter will only work with the sync
 context manager.
 
+Features:
+- Converts MCP tools to SmolAgents tools
+- Supports outputSchema for structured output (MCP spec 2025-06-18+)
+- Always uses "object" output_type for maximum flexibility
+- Handles text, image, and audio content types
+- Backwards compatible with tools without outputSchema
+
 Example Usage:
 >>> with MCPAdapt(StdioServerParameters(command="uv", args=["run", "src/echo.py"]), SmolAgentsAdapter()) as tools:
 >>>     print(tools)
+
```

Comment:
I feel this docstring is really too verbose compared to the rest of the repository. Can we keep it minimal but informative?